src/manual/Signatures/FileTypeMagic.md

```diff
@@ -31,3 +31,7 @@ Where:
 `min_flevel`: (optional) The minimum ClamAV engine that the file type signature works with. See the [FLEVEL reference](../../appendix/FunctionalityLevels.md) for details. To be used in the event that file type support has been recently added.
 
 `max_flevel`: (optional, requires `min_flevel` field, though the `min_flevel` number itself can be left empty) The maximum ClamAV engine that the file type signature works with. To be used in the event that file type support has been recently removed or replaced.
+
+> _Note_: It is likely that `magictype` `0` is insufficient to identify a file and so type `1` is often prefered.
```

Comment:
"prefered" => "preferred" for spelling, but perhaps it should be "may be necessary". I am curious to review this based on the runtime impact of type 0 versus 1 signatures in the matcher.